### DISCUSSION TITLE
请问能不能结合其他ui库的组件啊

### DISCUSSION THREAD

**Discussion by lovesL** [tolking/element-pro-components#471] [Q&A]
请问能不能结合其他ui库的组件啊
![image](https://github.com/tolking/element-pro-components/assets/27960630/70d14124-febe-408f-a0f6-21b24230acd2)
虽然这样用插槽能实现但是要单独绑定这个值 然后合到search-form 的里面 挺麻烦的 
有没有什么API 能渲染其他UI库的组件然后自己绑定值

**Comment by tolking** [comment]
想要支持 `v-model:value` 的方式绑定值？

倒是可以提供一个 api 实现自定义

**Reply by lovesL** [reply]
啥API了

**Reply by tolking** [reply]
#473 [demo](https://tolking.github.io/element-pro-components/zh-CN/components/form#%E9%85%8D%E7%BD%AE%E7%BB%84%E4%BB%B6%E7%BB%91%E5%AE%9A%E7%9A%84-v-model-%E5%8F%82%E6%95%B0)
